packages/templates/src/templates/contracts/ink/psp22/test/index.test.ts.hbs

```diff
@@ -1,7 +1,7 @@
 import { expect, use } from "chai";
 import chaiAsPromised from "chai-as-promised";
-import {{contract_name_pascal}}Factory from "./typedContract/constructors/{{contract_name}}";
-import {{contract_name_pascal}} from "./typedContract/contracts/{{contract_name}}";
+import {{contract_name_pascal}}Factory from "../../typedContracts/{{contract_name}}/constructors/{{contract_name}}";
```

Comment:
Not only PSP22 template, but also all other ink!&ask! test templates as well, such as flipper.

Reply:
oh, a mistake on flipper, but ask! I left out on purpose since I'll remove them in next PR.